thrift/shared.thrift

```diff
@@ -1087,6 +1108,8 @@ struct RegisterDomainRequest {
   50: optional bool emitMetric = true
   60: optional list<ClusterReplicationConfiguration> clusters
   70: optional string activeClusterName
+  // activeClusters is a map of region name to active cluster name for active-active domain
+  75: optional map<string, string> activeClustersByRegion
```

Comment:
This is <string, string> map because all we need is cluster name per region during domain registration. Initial failover versions will be assigned by domain handler.

Reply:
It's different from proto, which makes it complicated to create mappers.

Reply:
updated proto side to match this.